targets/ramips-mt76x8

```diff
@@ -124,6 +124,10 @@ device('xiaomi-mi-router-4a-100m-international-edition', 'xiaomi_mi-router-4a-10
 	factory = false,
 })
 
+device('xiaomi_mi-router-4a-100m-intl-v2', 'xiaomi_mi-router-4a-100m-intl-v2', {
```

Comment:
Is "xiaomi_mi-router-4a-100m-intl-v2" really the string printed by `lua -e 'print(require("platform_info").get_image_name())'`? Looking at the Device Tree, I would expect it to be "xiaomi-mi-router-4a-100m-international-edition-v2".

Reply:
So then it should be as following: :)

```suggestion
device('xiaomi-mi-router-4a-100m-international-edition-v2', 'xiaomi_mi-router-4a-100m-intl-v2', {
```